app/styles/templates/project/thankyou.scss

```diff
@@ -1,14 +1,27 @@
 .thankyou-container {
-  .icon, .header, .text, .project-link{
+  .icon, .header, .text, .project-link {
     width: 100%;
     text-align: center;
   }
 
-  .icon img {
-    display: inline-block;
+  .header, .text {
+    margin-bottom: 20px;
```

Comment:
I'd set margins as ems as well, 14px is the baseline multiplier so 20px would translate to "margin-bottom: 1.4em"